.circleci/make-deb-packages.sh

```diff
@@ -49,4 +49,16 @@ dpkg-buildpackage -us -uc -j16
 
 echo "Prepare the packages for storage..."
 mkdir -p packages/$PARAM_DIST/$PARAM_RELEASE/
-mv ../*.deb ../*.dsc packages/$PARAM_DIST/$PARAM_RELEASE/
+mv ../*.deb packages/$PARAM_DIST/$PARAM_RELEASE/
+
+ARCH=`uname -m`
+ARCH2=""
+if [ "$ARCH" = "x86_64" ]; then
+  ARCH2="amd64"
+else
+  ARCH2="arm64"
+fi
```

Comment:
```suggestion
if [ "`uname -m`" = "x86_64" ]; then
  ARCH="amd64"
else
  ARCH="arm64"
fi
```

Reply:
Applied!
I didn't use GitHub UI to apply your change because it needed to change the last line as well.